src/aesthetics.jl

```diff
@@ -370,7 +370,7 @@ function by_xy_group(aes::T, xgroup, ygroup,
                     if !applicable(convert, typeof(vals), staging[i, j])
                         T2 = eltype(vals)
                         if T2 <: Color T2 = Color end
-                        da = DataArray(T2, length(staging[i, j]))
+                        da = Array{Union{Missing,T2}}(length(staging[i, j]))
```

Comment:
This will also need an `undef`, or you can initialize with `missing`s.

Reply:
why didn't femtocleaner catch this?

Reply:
I've actually seen quite a few instances of FemtoCleaner not getting the `undef` thing.

Reply:
ditto.  is it worth submitting an issue to femtocleaner?

Reply:
Could be, yeah.